sdk/appconfiguration/azure-appconfiguration-provider/setup.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import sys
+import re
+import os.path
+from io import open
+from setuptools import find_packages, setup
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-appconfiguration-provider"
+PACKAGE_PPRINT_NAME = "App Configuration Provider"
+
+# a-b-c => a/b/c
+package_folder_path = PACKAGE_NAME.replace("-", "/")
+# a-b-c => a.b.c
+namespace_name = PACKAGE_NAME.replace("-", ".")
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, "_version.py"), "r") as fd:
+    version = re.search(
+        r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]', fd.read(), re.MULTILINE
+    ).group(1)
+
+if not version:
+    raise RuntimeError("Cannot find version information")
+
+with open("README.md", encoding="utf-8") as f:
+    readme = f.read()
+with open("CHANGELOG.md", encoding="utf-8") as f:
+    changelog = f.read()
+
+exclude_packages = [
+    "tests",
+    "tests.*",
+    "samples",
+    # Exclude packages that will be covered by PEP420 or nspkg
+    "azure",
```

Comment:
```suggestion
    "azure",
    "azure.appconfiguration"
```

Reply:
Source of the wonky import error you're seeing in `apistubgen` failures in `build` job.